### ISSUE TITLE
is it dead now?

### ISSUE THREAD

**Issue by michaelhmich** [Zren/plasma-applet-eventcalendar#348]
no update regarding google api in sight?
is there an alternative for google calendar?
thanks

**Comment by mansoormojo** [comment]
I have seen a couple forks that still, at least somewhat work for google calendar integration, of which are also still acknowledged by Zren and even seemingly supported by him.

**Comment by steelsnake** [comment]
Sadly it's absolutely dead now. As of two days ago even the previous working fork stopped working. I haven't been able to find a working fork either. Unfortunate, it was an excellent applet while it worked.

**Comment by boriseto** [comment]
@steelsnake I just used one of the forks and it pulled the data correctly (I was using the PIM before but I messed it up with switching DEs).

**Comment by steelsnake** [comment]
Mind mentioning which one...?

**Comment by boriseto** [comment]
> Mind mentioning which one...?

https://github.com/kanocz/plasma-applet-eventcalendar

**Comment by steelsnake** [comment]
Unfortunately I still can't get it to work. I deleted any old config (I think) and did the following steps, yet I get an error "Something went wrong" whenever I try to authorize access to my Google Account. Go is installed.

rm -rf ~/.local/share/plasma/plasmoids/org.kde.plasma.eventcalendar
sudo rm -rf /usr/share/plasma/plasmoids/org.kde.plasma.eventcalendar
git clone https://github.com/kanocz/plasma-applet-eventcalendar
cd plasma-applet-eventcalendar
./install

After that I add the widget, and get the rather nebulous "Something went wrong" error message.

Am I doing something wrong here? Missing a step maybe? Any way to debug this?

**Comment by boriseto** [comment]
I did the same steps
Maybe after the install (cause plasma crashed) I accidentally did the `update` command instead of `uninstall` (which just calls the install command anyway) and it just started working.
Sorry that I'm not too much help.

**Comment by sasa-tomic** [comment]
I forked the repo and fixed all issues I've hit so far. Feel free to give it a shot and to star the repo: https://github.com/sasa-tomic/plasma-applet-eventcalendar

**Comment by bursterburster** [comment]
> I forked the repo and fixed all issues I've hit so far. Feel free to give it a shot and to star the repo: https://github.com/sasa-tomic/plasma-applet-eventcalendar

Nice. Any chance you could enable the issues tab?
I’m stuck with installation:
```

┬─ burster@X250:~/Development
╰─> $ git clone https://github.com/sasa-tomic/plasma-applet-eventcalendar                                                                                                                                                                                            12:43:24
Klone nach 'plasma-applet-eventcalendar'...
remote: Enumerating objects: 9778, done.
remote: Counting objects: 100% (227/227), done.
remote: Compressing objects: 100% (81/81), done.
remote: Total 9778 (delta 159), reused 156 (delta 146), pack-reused 9551 (from 2)
Empfange Objekte: 100% (9778/9778), 2.26 MiB | 6.28 MiB/s, fertig.
Löse Unterschiede auf: 100% (6544/6544), fertig.

┬─ burster@X250:~/Development
╰─> $ cd plasma-applet-eventcalendar/                                                                                                                                                                                                                                12:43:27

┬─ burster@X250:~/Development/plasma-applet-eventcalendar [Git:master]
╰─> $ sh ./install                                                                                                                                                                                                                                                   12:43:33
Warning: Could not locate service type file kservicetypes5/plasma-applet.desktop, tried ("/home/burster/.local/share", "/usr/local/share", "/usr/share") and ":/kservicetypes5/plasma-applet.desktop" (:0, )
```

**Comment by gdfox** [comment]
I got the same error.

**Comment by sasa-tomic** [comment]
yes sorry @bursterburster @gdfox  I didn't get the ping earlier so only saw the message now. I enabled issues there.
